tests/basic_qp/test_basic_qp.h

```diff
@@ -993,24 +981,4 @@ static const char* test_basic_qp_warm_start()
 
   // Cleanup
   c_free(settings);
-
-  return 0;
-}
-
-
-static const char* test_basic_qp()
```

Comment:
All driver functions in `test_*.h` files are no longer needed. Already approved in `master`.